src/ContractEditor/plugins/withVariables.js

```diff
@@ -34,7 +34,15 @@ export const isEditable = (lockText, editor, event) => {
   if (!lockText || !editor.isInsideClause()) return true;
   const { selection } = editor;
   const textLength = Node.get(editor, editor.selection.focus.path).text.length;
+  const atEnd = (editor) => {
+    const textLength = Node.get(editor, editor.selection.focus.path).text.length;
```

Comment:
`textLength` is already defined on line 36

Reply:
👀